vulnfeeds/cves/versions.go

```diff
@@ -256,3 +271,30 @@ func CPEs(cve CVEItem) []string {
 	}
 	return cpes
 }
+
+// Parse a well-formed CPE string into a struct.
+func ParseCPE(cpe_str string) (*CPE, bool) {
+	if !strings.HasPrefix(cpe_str, "cpe:") {
+		return nil, false
+	}
+
+	cpe_fields := strings.Split(cpe_str, ":")
+
+	if len(cpe_fields) < 12 {
```

Comment:
Should this be 13 instead? cpe_fields[12] implies that there are 13 expected items given zero indexing.

Reply:
Sigh. Yeah. So sloppy of me :-(